_knitr/A Shiny app for Hong Kong District Councillors_20200801.md

```diff
@@ -1,50 +1,204 @@
+- Release to alpha testers to get early feedback.
+- More or our enhancements / spikes are listed here on [GitHub](https://github.com/Hong-Kong-Districts-Info/dashboard-hkdistrictcouncillors/issues)
+
+One of the things that we wanted to try out with this open-source project is to adhere to some DevOps best practices, yet unfortunately some of these were either easier to set up from the beginning, or require more time and knowledge (on our part) to set up. As we develop a V2 of this Shiny App and work on [other projects](https://hong-kong-districts-info.github.io/portfolio/), we hope to find the opportunity to implement more of the above features.
+
+## 🔥 Other features in the app
+
+There were also a number of features that we have implemented, but were not detailed in this post. For instance:
+
+- Adding a searchable DataTable with information on the District Councillors, with the **DT** package
+- Embedding a user survey within the Shiny app
+- Adding a tutorial to go through features of the Shiny app, using the **shinyhelper** package
+- Adding loader animations with **shinycssloaders**
+
+We will cover more of that detail in a Part 2 of this blog, so watch this space!
+    
+## 💪 Who is behind this?
+
+Multiple people contributed to this work. **Avision Ho** is a data scientist who wrote the majority of the Shiny app, and who was also [previously interviewed on this blog](https://martinctc.github.io/blog/data-chats-an-interview-with-avision-ho/). Avision is a co-author on this post. **Ocean Cheung** came up with the original idea of this app, and made it all possible with his knowledge and network with District Councillors. We would also like to credit **Justin Yim**, **Tiffany Chau**, and **Gabriel Tam** for their 
+feedback and advice on the scope and the direction of this app. We are currently working on a number of other projects, which you can find out more from our website: https://hong-kong-districts-info.github.io/.
+
+(Disclaimer! We are not affiliated to any political individuals nor movements. We are simply some people who'd like to contribute to society through code and open-source projects.)
+    
+## ✋ Want to get involved?
+
+We're looking for collaborators or reviewers, so please send us an email (hkdistricts.info@gmail.com), or comment down below if you are interested! We would also appreciate any feedback or questions, which you could either comment below or respond to our [in-app survey](https://hkdistricts-info.shinyapps.io/dashboard-hkdistrictcouncillors/).
+
+When we first started out, we were just a couple of people who wanted to learn and practise a new skill (e.g. building a Shiny app, implementing best practices), and wanted a meaningful open-source project that we could work on. Read more about [our Vision Statement here](https://hong-kong-districts-info.github.io/about/).
+
```

Comment:
Our Trello board is also public because we're entirely open-source!
```suggestion

You can also get an idea of things we are planning to work on through our Trello board [here](https://trello.com/b/n5l7DMS5/doing).
```